src/js/components/search/filters/location/PlaceOfPerformanceTypeahead.jsx

```diff
@@ -0,0 +1,175 @@
+/**
```

Comment:
Let's keep a consistent naming scheme of `components/..../xxx.jsx` being the immediate child of `containers/.../xxxContainer.jsx`